core-libs/setup/ssr/error-handling/multi-error-handlers/propagating-to-server-error-handler.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { TestBed } from '@angular/core/testing';
+import { FeatureConfigService } from '@spartacus/core';
+import { PROPAGATE_ERROR_TO_SERVER } from '../error-response/propagate-error-to-server';
+import { PropagatingToServerErrorHandler } from './propagating-to-server-error-handler';
+
+describe('PropagatingToServerErrorHandler', () => {
+  describe('default factories', () => {
```

Comment:
[MINOR]
I guess we don't need this wrapping `describe` anymore. Instead all can be in the main block `describe('PropagatingToServerErrorHandler'`. Would you agree?

Reply:
Thanks, good point!